package.json

```diff
@@ -45,27 +46,33 @@
     "**/__tests__/fixtures/"
   ],
   "private": true,
+  "lint-staged": {
+    "*.{js,jsx}": [
+      "eslint --fix",
+      "git add"
+    ],
+    "*.{json,md,scss,yaml,yml}": [
+      "prettier --write",
+      "git add"
+    ]
+  },
   "scripts": {
     "bootstrap": "npm-run-all -s check-versions lerna-prepare",
     "check-versions": "babel-node scripts/check-versions.js",
-    "format": "npm-run-all -p format-packages format-cache-dir format-www format-integration format-examples format-scripts format-markdown format-yaml",
-    "format-cache-dir": "prettier-eslint --write \"packages/gatsby/cache-dir/*.js\"",
-    "format-examples": "prettier-eslint --write \"examples/**/gatsby-node.js\" \"examples/**/gatsby-config.js\" \"examples/**/src/**/*.js\"",
-    "format-markdown": "prettier --write \"**/*.md\" --no-semi",
-    "format-packages": "prettier-eslint --write \"packages/*/src/**/*.js\"",
-    "format-scripts": "prettier-eslint --write \"scripts/**/*.js\"",
-    "format-www": "prettier-eslint --write \"www/*.js\" \"www/src/**/*.js\"",
-    "format-integration": "prettier-eslint --write \"integration-tests/**/src/**/*.js\" \"integration-tests/**/cypress/**/*.js\"",
-    "format-yaml": "prettier --write \"**/*.y?(a)ml\"",
+    "format": "npm-run-all -p format-packages format:js format:other",
+    "format:other": "yarn prettier --write",
```

Comment:
Can we do `npm run prettier --write` here? I'd like to minimize the amount of places we require yarn explicitly!

Reply:
I would say we could get rid of npm-run-all and use yarn everywhere since we have `yarn.lock` and no `package-lock.json`

Reply:
I disagree a lot with that! We want to keep barrier to entry low, so a change like that just makes that barrier higher and harder to get to for new contributors.

Reply:
That's invisible to the users and we already using yarn in the 'prebootstrap' and defined as engine..

Reply:
I can imagine a user running `npm run lint:other` and being confused that fails. In that scenario, we made it harder for that user to contribute, and I'm not sure if it was for a real benefit.

Reply:
Updated it